.gitignore

```diff
@@ -0,0 +1,6 @@
+#ignore files with iml extenion
+*.iml
+
+
+#Ignore everything under .idea folder, not elsewhere
```

Comment:
Seems, now you missed the "out" folder, which is being generated by IntelliJ as well when we run the main method.

Reply:
I will add it now, thank you.